lib/puppet/util/windows/service.rb

```diff
@@ -427,6 +451,23 @@ def set_startup_mode(service_name, startup_type)
     end
     module_function :set_startup_mode
 
+    def set_startup_mode_delayed(service_name, startup_type, delayed)
+      change = SERVICE_CONFIG_CHANGES.key(startup_type) 
+      delayed_start = SERVICE_DELAYED_AUTO_START_INFO.new
+      delayed_start[:fDelayedAutostart] = delayed
+      open_service(service_name, SC_MANAGER_CONNECT, SERVICE_CHANGE_CONFIG) do |service|
+        success = ChangeServiceConfig2W(
+          service,
+          change,
+          delayed_start,
+        )
+        if success == FFI::WIN32_FALSE
+          raise Puppet::Util::windows::Error.new(_(“Failed to update service delayed_auto_start configuration”))
```

Comment:
I think rubocop is complaining about lowercase `windows` and/or the smart quotes?